styles.css

```diff
@@ -0,0 +1,148 @@
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 400;
+}
+
+.ticket-container {
+    margin: 50px 0 0 50px;
+    width: 782px;
+    height: 273px;
+    background: #EFC684;
+    border-radius: 10px;
+    display: flex;
+}
+
+.main-section {
+    width: 289px;
+}
+
+.ticket-text {
+    text-transform: uppercase;
+    font-size: 55px;
+    margin: 26px 0 0 20px;
+}
+
+.ticket-details-container {
+    display: flex;
+    justify-content: space-evenly;
+    margin-right: 30px;
+}
+
+.ticket-details {
+    width: 82px;
+    height: 44px;
+    border: solid 1px black;
+    font-size: 20px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    margin-top: 36px;
+    margin-bottom: 39px;
+}
+
+.ticket-category {
+    text-transform: uppercase;
+}
+
+.movie-details-container {
+    display: flex;
+    flex-direction: column;
+    margin-right: 52px;
+    justify-content: space-around;
+    font-size: 20px;
+}
+
+.movie-details {
+    height: 23px;
```

Comment:
This is unnecessary. Rely on the content to expand the parent if possible

Reply:
it is unnecessary because this class is not existing anymore, I forgot to remove it - I have removed it now